GVFS/GVFS.Service/ProductUpgradeTimer.cs

```diff
@@ -23,7 +22,7 @@ public void Start()
         {
             if (!GVFSEnlistment.IsUnattended(this.tracer))
             {
-                TimeSpan startTime = TimeSpan.Zero;
+                TimeSpan startTime = TimeSpan.FromMinutes(1);

```

Comment:
What is the reason for this change?

Reply:
Originally - I added a 1 minute delay before GVFS service checked for updates so that I had time to attach the debugger to the service. I tried following the basic tricks to debug service startup, but I was not successful - I suspect I was not getting interactive prompts, even though I had set the bit to allow the service to interact with the desktop....

However, I thought this might not be a bad change in general, as we delaying the initial check by a minute has minimal (no?) downside, and allows us to attach a debugger if needed. It could be a positive to wait a minute so we don't check while starting up.

I can put this back, if people object.

Reply:
I don't see a problem with waiting a minute.